.github/workflows/release-please.yaml

```diff
@@ -35,15 +35,15 @@ jobs:
         uses: actions/checkout@v3
       # Create SBOM
       - name: Generate SBOM
-        uses: CycloneDX/gh-gomod-generate-sbom@v2
+        uses: CycloneDX/gh-gomod-generate-sbom@v1
         with:
           version: v1
```

Comment:
It's already configured to use version 1.x.

Reply:
This version is not refering `gh-gomod-generate-sbom` but the underlying `cyclonedx-gomod` version